Example/Auth/Sample/MainViewController+OAuth.m

```diff
@@ -31,14 +31,16 @@ - (StaticContentTableViewSection *)oAuthSection {
                                        action:^{ [weakSelf signInGoogleHeadfulLite]; }],
     [StaticContentTableViewCell cellWithTitle:@"Sign in with Microsoft"
                                        action:^{ [weakSelf signInMicrosoftHeadfulLite]; }],
-    [StaticContentTableViewCell cellWithTitle:@"Sign In with GitHub"
+    [StaticContentTableViewCell cellWithTitle:@"Sign In with GitHub (access token)"
```

Comment:
Can you reorder these providers to align with the order of the doc? https://firebase.google.com/docs/auth/ios/microsoft-oauth

Reply:
Done